packages/client/scripts/default-index.d.ts

```diff
@@ -38,11 +38,11 @@ export declare class PrismaClientExtends<
   ExtArgs extends runtime.Types.Extensions.Args = runtime.Types.Extensions.DefaultArgs,
 > {
   $extends: { extArgs: ExtArgs } & (<
-    Q extends runtime.Types.Extensions.UserArgs['query'] = {},
     R extends runtime.Types.Extensions.UserArgs['result'] = {},
     M extends runtime.Types.Extensions.UserArgs['model'] = {},
+    Q extends runtime.Types.Extensions.UserArgs['query'] = {},
     C extends runtime.Types.Extensions.UserArgs['client'] = {},
-    Args extends runtime.Types.Extensions.Args = runtime.Types.Extensions.InternalArgs<Q, R, M, C>,
+    Args extends runtime.Types.Extensions.Args = runtime.Types.Extensions.InternalArgs<R, M, {}, C>,
```

Comment:
This is the fix as well; I changed the order of the generics, which is important when Accelerate ships its compiled types, because TypeScript will inline Accelerate's inferred types into expanded `InternalArgs<R, M, {}, C>`. In other words, `R, M, Q, C` get actual type values inlined. Then a user with using Accelerate on Prisma Client 4.16.0 would have a client that understands the order differently, so this was a subtle breaking change.